Test/dafny2/SnapshotableTrees.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 4 %dafny /compile:4 /dprint:"%t.dprint" "%s" > "%t"
```

Comment:
I should probably switch this back to 4 and add back in the execution output.

Reply:
Yup :)